src/main/java/com/coveros/selenified/utilities/Reporter.java

```diff
@@ -32,24 +32,29 @@
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
 import com.openhtmltopdf.pdfboxout.PdfRendererBuilder;
+import org.testng.*;
 import org.testng.log4testng.Logger;
+import org.testng.xml.XmlSuite;
 
 import java.io.*;
 import java.nio.file.FileSystems;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.text.SimpleDateFormat;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
+import java.util.function.Function;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.function.Function'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)